src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/BootJsonBuilderHelper.cs

```diff
@@ -18,7 +18,11 @@ public class BootJsonBuilderHelper(TaskLoggingHelper Log, string DebugLevel, boo
         ];
 
         private static readonly string[] extraMultiThreadedCoreAssemblyName = [
-            "System.Threading.Channels"
+            "System.Threading.Channels",
```

Comment:
We have unit test failing with `Could not load file or assembly System.Threading` when the timing of the downloads is in favorable order. 

I'm not sure if I need to really provide all dependencies of `System.Threading.Channels` but I do it just to be on the safe side.